.github/workflows/axl-lab1.yml

```
@@ -0,0 +1,124 @@
+  push:
+    branches:
+      - axl-3.0.6
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: axl-lab1
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)